README.md

```diff
@@ -24,3 +24,85 @@ You should run the spawn server in the same container as the program that uses t
 ## Build
 
 Just execute `cargo build --release` to build the spawn server. 
+
+## `posix_spawn(2)` mode
+
+The spawn server allows the execution of `posix_spawn(2)` in the stead of a requesting client.
+In particular, this substitute `posix_spawn(2)` aims to be a 
```

Comment:
This sentence seems to be incomplete.

Also, maybe, add a little problem description - spawned child is owned by spawn_server and that might cause problems since the parent might wait for the child to terminated. Hence, we need to "reparent" the spawned child...